apps/worker/src/app/workflow/services/trigger-processor-queue.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@nestjs/common';
+const nr = require('newrelic');
+import {
+  TriggerQueueService,
+  PinoLogger,
+  storage,
+  Store,
+  TriggerEvent,
+  TriggerEventCommand,
+} from '@novu/application-generic';
+
+@Injectable()
+export class TriggerProcessorQueueService extends TriggerQueueService {
```

Comment:
the trigger processor queue, which creates the worker and calls the trigger event use-case